### ISSUE TITLE
Can't Kill in Certain Territory? 

### ISSUE THREAD

**Issue by MoeXIV** [drtshock/HCFProblems#18]
So, I've been doing a bunch of bug tests with players, just to ensure I have no issues when we fully release. I found that if you have no Faction, you cannot PvP in someone elses Territory. When you try to hit them, it just sends you a blank line in chat, and doesn't let you pvp. I've looked through every file and nothing about PvP comes up.

----

Also, people with no Faction, should have a red name tag due to the fact in HCF, everyone is an enemy, Faction or not.
http://prntscr.com/6i03sk

**Comment by nsporillo** [comment]
@MoeXIV For the second part of your issue, this was addressed [here](https://github.com/drtshock/HCFProblems/issues/12). 

In your lang file, change:
    default-prefix: '{relationcolor}[{faction}] &r'
to:
    default-prefix: '{relationcolor}'

This should result in: 
![2015-03-17_01 58 13](https://cloud.githubusercontent.com/assets/1480197/6682102/327f1bd2-cc49-11e4-9b33-505e69226fea.png)

Make sure default-prefixes in the config file is set to true. 

If you're still having issues, let me know.

**Comment by Doxify** [comment]
People that are not in a faction have a white name, we'd like for you to change that to red.

**Comment by nsporillo** [comment]
Could just go into the conf.json and change the color for neutral factions to the color you have for enemy factions.

This 'hack' should suffice until I can thoroughly test the changes to a utility method with a huge scope. (Lots of stuff relies on this method, so changing it may introduce bugs elsewhere)

**Comment by Doxify** [comment]
![8tarryz](https://cloud.githubusercontent.com/assets/9636316/6703979/20c79628-cd00-11e4-8c53-eb8d06372d81.png)

That doesn't work

**Comment by nsporillo** [comment]
@Doxify Oh, sorry about that.

Give me a few minutes to test out this experimental change I just made. Hopefully it'll solve this issue and introduce no other bugs (relation utils are used a lot, so I hope not)

**Comment by nsporillo** [comment]
Going to set this to waiting, since i'm going to sleep and still havent solved the 2nd issue yet. 

Could not reproduce the pvp issue on my own. It's late thought. I had to open 3 different clients, one of them cracked since I only own two accounts, just so I could try and figure out whats wrong. It's working as expected for me, so a video or more explanation would be fantastic!

Progress on issues has ceased as it is 3:24am, whatever I change / test just doesnt work with diminished level of focus. I'll be on tomorrow when I wake to handle issues :)

**Comment by drtshock** [comment]
https://github.com/drtshock/HCFactions/blob/dtr/src/main/resources/config.yml#L26

Change the default relation.

**Comment by Doxify** [comment]
Page not found ^

**Comment by Doxify** [comment]
Guessing you're talking about this, as you can see it still doesn't work.

![iliecbl](https://cloud.githubusercontent.com/assets/9636316/6710774/35c84d98-cd40-11e4-8809-0ea9c67e078f.png)

**Comment by drtshock** [comment]
Which person are you?

**Comment by MoeXIV** [comment]
He is LizardSquad. If you're not in a Faction, you have no nametag to anyone holding the tablist. With or without a Faction, you should have the "default" colored tag (Defined in your config.yml)

**Comment by drtshock** [comment]
But Sainly has a red tag as he is an enemy. I fail to see what isn't working.

**Comment by Doxify** [comment]
We want EVERYONE who is not in a faction to have a red tag.

Just give the wilderness faction a red tag.

**Comment by drtshock** [comment]
Why would you show up as hostile to yourself though?

**Comment by Doxify** [comment]
Correct, let me rephrase.

Make everyone's name who is not in a faction red for everyone else but green for them.
